### ISSUE TITLE
dhcpd: no such user: dhcpd

### ISSUE THREAD

**Issue by L1ghtn1ng** [opnsense/core#889]
I randomly lost DHCP as I think dhcp crashed so I pulled the power to the firewall to reboot it upon looking at the log this is what I found. This is I wonder also due to #888

**Comment by AdSchellevis** [comment]
@L1ghtn1ng I see them over here as well, but don't have issues with dhcp. It looks like hotplug tries to start dhcp before it should, then fails and then later it starts without issues. 

It looks harmless, but would be better to fix at some point.
Maybe we should exit  /usr/local/etc/rc.linkup while booting, but that needs further investigation.

**Comment by AdSchellevis** [comment]
Forgot to mention, its not related to the other message.

**Comment by fichtner** [comment]
This sounds like a pkg database corruption.

Does this work?

    # pkg info isc-dhcp43-server

**Comment by L1ghtn1ng** [comment]
@fichtner Yes that does work. Please see output

```
root@OPNsense:~ # pkg info isc-dhcp43-server
isc-dhcp43-server-4.3.3P1_1
Name           : isc-dhcp43-server
Version        : 4.3.3P1_1
Installed on   : Thu Mar 17 22:54:38 2016 GMT
Origin         : net/isc-dhcp43-server
Architecture   : freebsd:10:x86:64
Prefix         : /usr/local
Categories     : net
Licenses       : ISCL
Maintainer     : zi@FreeBSD.org
WWW            : https://www.isc.org/products/DHCP/
Comment        : ISC Dynamic Host Configuration Protocol server
Options        :
	BIND_SYMBOLS   : off
	IPV6           : on
	LDAP           : off
	LDAP_SSL       : on
	PARANOIA       : on
Annotations    :
	repo_type      : binary
	repository     : OPNsense
Flat size      : 3.97MiB
Description    :
The ISC Dynamic Host Configuration Protocol Distribution provides a
freely redistributable reference implementation of all aspects of the
DHCP protocol, through a suite of DHCP tools:

  * A DHCP server (this port)
  * A DHCP client
  * A DHCP relay agent

WWW: https://www.isc.org/products/DHCP/
```

**Comment by fichtner** [comment]
What's the contents of /etc/group ? If dhcpd (not _dhcpd) isn't in there, running the following should bring it back:

    # /usr/local/etc/rc.recover

**Comment by L1ghtn1ng** [comment]
@fichtner Sorry for taking so long to get back to this. This is the output

```
wheel:*:0:root
daemon:*:1:
kmem:*:2:
sys:*:3:
tty:*:4:
operator:*:5:root
mail:*:6:
bin:*:7:
news:*:8:
man:*:9:
games:*:13:
ftp:*:14:
staff:*:20:
sshd:*:22:
smmsp:*:25:
mailnull:*:26:
guest:*:31:
bind:*:53:
unbound:*:59:
proxy:*:62:
authpf:*:63:
_pflogd:*:64:
_dhcp:*:65:
uucp:*:66:
dialer:*:68:
network:*:69:
audit:*:77:
www:*:80:
hast:*:845:
nogroup:*:65533:
nobody:*:65534:
dhcpd:*:136:
_relayd:*:913:
squid:*:100:
admins:*:1999:root
```

**Comment by fichtner** [comment]
It looks fine. Is DHCP back to normal at this moment?

**Comment by L1ghtn1ng** [comment]
Not sure as I checked after upgrading from 16.1.10 to 16.1.11_1 best to monitor it for a bit

**Comment by fichtner** [comment]
This was fixed in 16.1.13, also see:

https://forum.opnsense.org/index.php?topic=3082
